gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -128,15 +115,30 @@ fluid.defaults("gpii.flowManager.local", {
             }
         }
         */
-        // TODO: After FLUID-6148 is implemented, directly saving "defaultLifecycleInstructions" into options should
-        // be replaced by expressing {that}.defaultLifecycleInstructions as an expander in the component's options
-        "userLogonHandling.defaultLifecycleInstructions": {
-            record: "{that}.options.defaultLifecycleInstructions",
-            target: "{that gpii.lifecycleManager.userLogonHandling.stateChangeHandler}.options.defaultLifecycleInstructions"
+        defaultSettingsDataPromise: "@expand:fluid.promise()"
+    },
+    contextAwareness: {
+        suppressHttpEndpoints: {
+            checks: {
+                addHttpEndpointsOption: {
+                    contextValue: "{gpii.flowManager.local}.options.suppressHttpEndpoints",
+                    gradeNames: null
+                }
+            },
+            defaultGradeNames: "gpii.flowManager.local.requestHandlers"
+        }
+    },
+    distributeOptions: {
+        // Distribute the grade to handle user logon requests.
+        // In all-in-local config, the handler is "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler".
+        // In untrusted config, the handler should be "gpii.lifecycleManager.untrusted.stateChangeHandler".
+        "lifecycleManager.loginRequestHandler": {
+            record: "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler",
+            target: "{that gpii.lifecycleManager.loginRequest}.options.gradeNames"
         },
-        "userLogonHandling.defaultSnapshot": {
-            record: "{that}.options.defaultSnapshot",
-            target: "{that gpii.lifecycleManager.userLogonHandling.stateChangeHandler}.options.defaultSnapshot"
+        "userLogonHandling.defaultSettingsDataPromise": {
+            record: "{gpii.flowManager.local}.defaultSettingsDataPromise",
+            target: "{that gpii.lifecycleManager.userLogonHandling.stateChangeHandler}.options.defaultSettingsDataPromise"
```

Comment:
Yes, I see this now - for consistency, send this into options.members rather than options